src/base_event.cpp

```diff
@@ -105,6 +116,7 @@ void MainFrame::OnBrowse(std::string& filetype, wxTextCtrl* textbox){
   }
   textbox->SetLabel(openFileDialog.GetPath());
   //load automatically
+  enableGuiElements(false);

```

Comment:
> Much simpler and cleaner to just call OnLoadFiles() directly. For that you also need to pass the wxCommandEvent& argument from the two OnBrowse functions.

I was not the one implementing the automatic file loading. I didn't change much of the code as it was working as intended (aside from this line) but I will look into it.

> While we're on this topic, the import should not happen before there are values in both text fields. Otherwise I get an error after I select a radius file. Something like:
> 
> ```c++
> if (atom_path == "" || radius_path == ""){
>   OnLoadFiles(event);
> }
> ```

I agree, I saw that minor issue directly and thought I would fix it later. Another possible fix would be to only import radii file after browsing the radii file and only import structure file after browsing the structure file (but still import both upon clicking on reload).

With your proposed fix, one minor issue would be that if someone loads an invalid structure file, he gets the error, then loads the radius file, he gets the error of the structure file again which seems redundant.

Which solution do you prefer?

Reply:
> I agree, I saw that minor issue directly and thought I would fix it later. Another possible fix would be to only import radii file after browsing the radii file and only import structure file after browsing the structure file (but still import both upon clicking on reload).

Previously, this would have been a more complicated fix, because the atom file import function required the radius map. Has this been changed now, so that both of the imports work independently (by that I mean `readRadiiAndAtomNumFromFile()` and `readAtomsFromFile()`)? If so, I'd prefer to load each file separately on separate browses.

Reply:
Ok, I'll work on that and implement it in the main branch.